apps/web/src/components/Menu/LogoutModal.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  Button,
+  Heading,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+
+import { useColor } from "../../styles/useColor";
+import { persistor } from "../../utils/persistor";
+import { ModalCloseButton } from "../ModalCloseButton";
+
+export const LogoutModal = () => {
+  const color = useColor();
+  const handleLogout = () => {
+    persistor.pause();
+    localStorage.removeItem("persist:accounts");
```

Comment:
and "persist:root" too, but I bet there is a ready to go function already that does handle all of that